tests/test_LSODA.cpp

```
@@ -171,28 +348,24 @@ int main(int argc, const char* argv[])
-    test_github_system();
-    end = chrono::steady_clock::now();
-    cout << "|| Time taken (us)="
-         << chrono::duration_cast<chrono::nanoseconds>(end - begin).count()
-            / 1000
-         << endl;
+    auto run = [](const char* name, auto fn) {
+        auto begin = chrono::steady_clock::now();
+        int rc = fn();
+        auto end = chrono::steady_clock::now();
+        if (rc != 0)
+            throw runtime_error(string(name) + " returned non-zero: " + to_string(rc));
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)